pynucastro/networks/base_cxx_network.py

```diff
@@ -134,11 +134,11 @@ def _write_network(self, odir=None):
         # Copy any tables in the network to the current directory
         # if the table file cannot be found, print a warning and continue.
         for tr in self.tabular_rates:
-            tdir = os.path.dirname(tr.rfile_path)
-            if tdir != os.getcwd():
-                tdat_file = os.path.join(tdir, tr.table_file)
-                if os.path.isfile(tdat_file):
-                    shutil.copy(tdat_file, odir or os.getcwd())
+            tdir = tr.rfile_path.parent
```

Comment:
`rfile_path` might be relative, which would break the comparison with `Path.cwd()`:
```suggestion
            tdir = tr.rfile_path.resolve().parent
```